src/main/java/com/eternalcode/discordapp/review/GitHubReviewService.java

```diff
@@ -45,22 +49,27 @@ public String createReview(Guild guild, String url, JDA jda) {
         }
     }
 
-    public boolean checkPullRequestTitle(String url) throws IOException {
+    public boolean checkPullRequestTitle(GitHubPullRequest url) throws IOException {
         String pullRequestTitleFromUrl = GitHubReviewUtil.getPullRequestTitleFromUrl(url, this.discordAppConfig.githubToken);
 
         return GitHubReviewUtil.isPullRequestTitleValid(pullRequestTitleFromUrl);
     }
 
-    public long createReviewForumPost(Guild guild, String url) throws IOException {
-        String pullRequestTitleFromUrl = GitHubReviewUtil.getPullRequestTitleFromUrl(url, this.discordAppConfig.githubToken);
+    public long createReviewForumPost(Guild guild, GitHubPullRequest pullRequest) throws IOException {
+        String pullRequestTitleFromUrl = GitHubReviewUtil.getPullRequestTitleFromUrl(pullRequest, this.discordAppConfig.githubToken);
         ForumChannel forumChannel = guild.getForumChannelById(this.discordAppConfig.reviewSystem.reviewForumId);
 
-        MessageCreateData createData = MessageCreateData.fromContent(url);
-        return forumChannel.createForumPost(pullRequestTitleFromUrl, createData).setName(url).complete().getThreadChannel().getIdLong();
+        MessageCreateData createData = MessageCreateData.fromContent(pullRequest.toUrl());
+
+        return forumChannel.createForumPost(pullRequestTitleFromUrl, createData)
+                .setName(pullRequest.toUrl())
```

Comment:
Tutaj akurat bym ustawił nazwę na GH-{NUMER_PR} {NAZWA_PR}, aby wyglądało to schludniej.

Reply:
Unrelated, zrób issue